docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -783,11 +783,11 @@ directory and build the example with the following option (replace
 `nrf52840dk_nrf52840` with your board name):
 
 ```
-$ west build -b nrf52840dk_nrf52840 -- -DCONFIG_CHIP_FACTORY_DATA=y -DCONFIG_CHIP_FACTORY_DATA_BUILD=y
+$ west build -b nrf52840dk_nrf52840 -- -DCONFIG_CHIP_FACTORY_DATA=y -DSB_CONFIG_MATTER_FACTORY_DATA_GENERATE=y
 ```
 
-Alternatively, you can also add `CONFIG_CHIP_FACTORY_DATA_BUILD=y` Kconfig
-setting to the example's `prj.conf` file.
+Alternatively, you can also add `SB_CONFIG_MATTER_FACTORY_DATA_GENERATE=y`
+Kconfig setting to the example's `prj.conf` file.
```

Comment:
are you sure it can be added to `prj.conf` ? In ncs SB_CONFIG_* configs are added to `sysbuild.conf`

Reply:
Yes, you are right. This change was propagated from downstream, so I think this should be fixed in NCS docs as well.